src/server/helpers/emailHelper.js

```diff
@@ -0,0 +1,38 @@
+const nodemailer = require('nodemailer');
+const smtpTransport = require('nodemailer-smtp-transport');
+
+/**
+ * @function sendMail
+ * @param Object
+ * @param function
+ */
+const sendMail = (emailBody, callback) => {
+  const transporter = nodemailer.createTransport(smtpTransport({
+    service: 'gmail',
```

Comment:
Can we add some more to this email. Make some good HTML template for it. It's too plain

Reply:
Done